### PR TITLE
kup-text-field [inputType="number"]: could set nr of integer digits and decimal digits

### PR DESCRIPTION
**Pull request by pasere-smeup**

### CODE REVIEW

File: packages/ketchup/src/managers/kup-math/kup-math.ts [hunk 1]
```diff
@@ -137,9 +142,90 @@ export class KupMath {
...
+     * @returns current group separator, by locale
+     */
+    groupSeparator() {
+        const numberWithGroupAndDecimalSeparator = 1000.1;
+        return Intl.NumberFormat(this.locale)
+            .formatToParts(numberWithGroupAndDecimalSeparator)
+            .find((part) => part.type === 'group').value;
+    }
+
+    matchNumericValueWithOptions(
```
Comment (lucafoscili) [inline packages/ketchup/src/managers/kup-math/kup-math.ts:188]: missing JSDoc description

File: packages/ketchup/src/managers/kup-math/kup-math.ts [hunk 2]
```diff
@@ -126,8 +127,12 @@ export class KupMath {
      * @param {string} format - Desired format. Defaults to '0,0.00' (i.e.: 2,000,000.51).
      * @returns {string} Formatted number.
```
Comment (lucafoscili) [inline packages/ketchup/src/managers/kup-math/kup-math.ts:128]: JSDoc: add the new argument

**Review by lucafoscili — APPROVED** [review]

File: packages/ketchup/src/managers/kup-math/kup-math.ts (hunk 2 above)
Comment (pasere-smeup) [reply packages/ketchup/src/managers/kup-math/kup-math.ts:128]: ok, done

File: packages/ketchup/src/managers/kup-math/kup-math.ts (hunk 1 above)
Comment (pasere-smeup) [reply packages/ketchup/src/managers/kup-math/kup-math.ts:188]: ok, done

**Pull request merged**